src/coin_test/backtest/trade.py

```diff
@@ -1,25 +1,23 @@
 """Define the Trade class."""
 
-import datetime as dt
-
 from ..util import AssetPair, Side
 
 
 class Trade:
     """Store the details of a trade."""
 
     def __init__(
-        self, asset_pair: AssetPair, side: Side, price: float, timestamp: dt.datetime
+        self, asset_pair: AssetPair, side: Side, amount: float, price: float
     ) -> None:
         """Initialize a Trade object.
 
         Args:
             asset_pair: The asset pair being traded
             side: The direction of the trade
             price: The price per share of the asset
-            timestamp: When the trade takes place
+            amount: The number of shares of the asset
```

Comment:
nit: amount should come before price in the docstring

Reply:
Done.